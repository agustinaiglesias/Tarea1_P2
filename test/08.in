definirFecha 10/02/2022
fechaActual
fechaActualExtendida
reiniciarEstructuras
siguienteDia
aumentarDias 20
fechaActual
fechaActualExtendida
definirFecha 01/02/2020
fechaActual
fechaActualExtendida
Fin