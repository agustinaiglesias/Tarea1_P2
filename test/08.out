1>Fecha definida en forma exitosa.
2>La fecha actual es 10/2/2022.
3>La fecha actual es 10 de Febrero de 2022.
4>Estructuras reiniciadas.
5>La fecha del sistema no se ha definido.
6>La fecha del sistema no se ha definido.
7>La fecha del sistema no se ha definido.
8>La fecha del sistema no se ha definido.
9>Fecha definida en forma exitosa.
10>La fecha actual es 1/2/2020.
11>La fecha actual es 1 de Febrero de 2020.
12>Fin.
